test/server/lib/download/video.spec.js

```diff
@@ -438,31 +434,19 @@ describe(MODULE_ID, function () {
 					user: USER
 				});
 
-				dl.on('end', async () => {
-					extractedFiles = await decompress(filename, extractDir);
-
-					article = extractedFiles.find(item => item.path.endsWith(content.transcriptExtension));
-					captions = extractedFiles.find(item => item.path === path.basename(url.parse(content.captions[0].url).pathname));
-					media = extractedFiles.find(item => item.path.endsWith(content.download.extension));
-
-					done();
-				});
 
 				dl.pipe(fs.createWriteStream(filename));
-
-				dl.appendAll().then(() => {});
-			});
-
-			it('article', function() {
-				expect(article.data.equals(fs.readFileSync(path.resolve(`${FIXTURES_DIRECTORY}/article.${content.transcriptExtension}`)))).to.be.true;
-			});
-
-			it('captions', function() {
-				expect(captions.data.toString('utf8')).to.equal(dl.captionFiles[0].file);
+				await dl.appendAll();
 			});
 
-			it('media', function() {
-				expect(media.data.equals(fs.readFileSync(path.resolve(`${FIXTURES_DIRECTORY}/video-small.mp4`)))).to.be.true;
+			xit('media', function() {
```

Comment:
**Question**: What's the benefit of using xit? Do you have a link to documentation?

Reply:
xit is for skip the test , i forgot to remove it. it was for development. thanks!